<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCAGenerationClass</class>
 <widget class="QDialog" name="TCAGenerationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCAGeneration</string>
  </property>
  <widget class="QPushButton" name="pBtnGenerate">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>590</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>591</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Facility</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>524</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="lbSearchingradius">
       <property name="text">
        <string>Nearest point searching radius:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lEditSearchingradius"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbUnifiedcutoff">
       <property name="text">
        <string>Unified cut-off</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lEditUnifiedcutoff"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="pBtnOpenfacility">
       <property name="text">
        <string>Open Facility</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lEditFacilitypath"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>80</y>
      <width>277</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QLabel" name="lbFacilitytype">
       <property name="text">
        <string>Facility type:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="rBtnPoint">
         <property name="text">
          <string>point-based</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rBtnMpoint">
         <property name="text">
          <string>multiple point-based</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>130</y>
      <width>457</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lbFacilityid">
         <property name="text">
          <string>ID </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbBoxFacilityid"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lbFacilitylabel">
         <property name="text">
          <string>Label</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbBoxFacilitylabel"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lbFacilitycutoff">
         <property name="text">
          <string>Cut-off</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbBoxFacilitycutoff"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>80</y>
      <width>202</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_18">
     <item>
      <widget class="QLabel" name="lbCutofftype">
       <property name="text">
        <string>Cut-off type:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QRadioButton" name="rBtnCutoffunified">
         <property name="text">
          <string>unified</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rBtnCutoffidv">
         <property name="text">
          <string>individual</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>591</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>294</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QCheckBox" name="chkBoxAccedges">
       <property name="text">
        <string>acceessible edges</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="chkBoxCatchment">
       <property name="text">
        <string>catchment areas</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>591</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Network</string>
   </property>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QPushButton" name="pBtnOpenet">
       <property name="text">
        <string>Open Network </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lEditNetpath">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>144</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QRadioButton" name="rBtnNetundirected">
       <property name="text">
        <string>undirected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rBtnNetdirected">
       <property name="text">
        <string>directed</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>548</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="lbNetid">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbBoxNetid"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="lbNetsource">
         <property name="text">
          <string>Source</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QComboBox" name="cbBoxNetsource"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="lbNettarget">
         <property name="text">
          <string>Target</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbBoxNettarget"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="lbNetDirection">
         <property name="text">
          <string>Direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbBoxNetdirection"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbNettype">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>80</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Networkd type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Generation direction:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>158</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_19">
     <item>
      <widget class="QRadioButton" name="rBtnFromfacility">
       <property name="text">
        <string>from-facility</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rBtnTofacility">
       <property name="text">
        <string>to-facility</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TCAGeneration.qrc"/>
 </resources>
 <connections/>
</ui>
